content/en/docs/setup/install/multicluster/multi-primary/index.md

```diff
@@ -101,16 +155,66 @@ You can now [verify the installation](/docs/setup/install/multicluster/verify).
 
 ## Cleanup
 
+Uninstall Istio from both `cluster1` and `cluster2` using `istioctl` or `helm`.
+
+{{< tabset category-name="multicluster-uninstall-type-cluster-1" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
 1. Uninstall Istio in `cluster1`:
 
-    {{< text syntax=bash snip_id=none >}}
-    $ istioctl uninstall --context="${CTX_CLUSTER1}" -y --purge
-    $ kubectl delete ns istio-system --context="${CTX_CLUSTER1}"
-    {{< /text >}}
+{{< text syntax=bash snip_id=none >}}
+$ istioctl uninstall --context="${CTX_CLUSTER1}" -y --purge
+$ kubectl delete ns istio-system --context="${CTX_CLUSTER1}"
+{{< /text >}}
```

Comment:
```suggestion
    {{< text syntax=bash snip_id=none >}}
    $ istioctl uninstall --context="${CTX_CLUSTER1}" -y --purge
    $ kubectl delete ns istio-system --context="${CTX_CLUSTER1}"
    {{< /text >}}
```

Reply:
This still needs to be fixed

Reply:
I was getting a lint error when I tried indenting this. Also, I realized the numbers in the list are not showing up in the proper order: https://deploy-preview-15459--preliminary-istio.netlify.app/latest/docs/setup/install/multicluster/primary-remote/#cleanup. Trying to find a workaround.

Reply:
They're not showing in the proper order because the block does need to be indented. There shouldn't be a lint error with that because that's how it is done throughout the docs.

Reply:
@dhawton still running into with issues for this. Think this might be an issue with the numbering within a tab block? 

I added the indentation but numbers are still not in the correct order: https://deploy-preview-15459--preliminary-istio.netlify.app/latest/docs/setup/install/multicluster/multi-primary/#cleanup. 

```
    1. Delete Istio Helm installation from `cluster1`:

    {{< text syntax=bash >}}
    $ helm delete istiod -n istio-system --kube-context "${CTX_CLUSTER1}"
    $ helm delete istio-base -n istio-system --kube-context "${CTX_CLUSTER1}"
    {{< /text >}}
```

If I un-indent the numbers (like below), then the formatting still shows up incorrectly:

```
1. Delete Istio Helm installation from `cluster1`:

    {{< text syntax=bash >}}
    $ helm delete istiod -n istio-system --kube-context "${CTX_CLUSTER1}"
    $ helm delete istio-base -n istio-system --kube-context "${CTX_CLUSTER1}"
    {{< /text >}}
```

Reply:
Because you indented the numbers... you're supposed to only indent the lines *under* the number that are part of that number... in this case the text block. My suggestion only referenced the text block, not the number itself.

Reply:
Even when I unindented the numbers the error still persisted (see above comment) but I will try again.

Reply:
Formatting still messed up:

![image](https://github.com/user-attachments/assets/7e33e9dd-f24f-4163-a7aa-8147f55ef0fa)

Reply:
Took @bleggett's suggestion and removed the numbers altogether.